include/core/jbackend.h

```diff
@@ -61,79 +66,365 @@ struct JBackend
 	{
 		struct
 		{
-			gboolean (*backend_init) (gchar const*);
-			void (*backend_fini) (void);
+			gboolean (*backend_init)(gchar const*);
+			void (*backend_fini)(void);
 
-			gboolean (*backend_create) (gchar const*, gchar const*, gpointer*);
-			gboolean (*backend_open) (gchar const*, gchar const*, gpointer*);
+			gboolean (*backend_create)(gchar const*, gchar const*, gpointer*);
+			gboolean (*backend_open)(gchar const*, gchar const*, gpointer*);
 
-			gboolean (*backend_delete) (gpointer);
-			gboolean (*backend_close) (gpointer);
+			gboolean (*backend_delete)(gpointer);
+			gboolean (*backend_close)(gpointer);
 
-			gboolean (*backend_status) (gpointer, gint64*, guint64*);
-			gboolean (*backend_sync) (gpointer);
+			gboolean (*backend_status)(gpointer, gint64*, guint64*);
+			gboolean (*backend_sync)(gpointer);
 
-			gboolean (*backend_read) (gpointer, gpointer, guint64, guint64, guint64*);
-			gboolean (*backend_write) (gpointer, gconstpointer, guint64, guint64, guint64*);
-		}
-		object;
+			gboolean (*backend_read)(gpointer, gpointer, guint64, guint64, guint64*);
+			gboolean (*backend_write)(gpointer, gconstpointer, guint64, guint64, guint64*);
+		} object;
 
 		struct
 		{
-			gboolean (*backend_init) (gchar const*);
-			void (*backend_fini) (void);
+			gboolean (*backend_init)(gchar const*);
+			void (*backend_fini)(void);
 
-			gboolean (*backend_batch_start) (gchar const*, JSemanticsSafety, gpointer*);
-			gboolean (*backend_batch_execute) (gpointer);
+			gboolean (*backend_batch_start)(gchar const*, JSemanticsSafety, gpointer*);
+			gboolean (*backend_batch_execute)(gpointer);
 
-			gboolean (*backend_put) (gpointer, gchar const*, gconstpointer, guint32);
-			gboolean (*backend_delete) (gpointer, gchar const*);
-			gboolean (*backend_get) (gpointer, gchar const*, gpointer*, guint32*);
+			gboolean (*backend_put)(gpointer, gchar const*, gconstpointer, guint32);
+			gboolean (*backend_delete)(gpointer, gchar const*);
+			gboolean (*backend_get)(gpointer, gchar const*, gpointer*, guint32*);
 
-			gboolean (*backend_get_all) (gchar const*, gpointer*);
-			gboolean (*backend_get_by_prefix) (gchar const*, gchar const*, gpointer*);
-			gboolean (*backend_iterate) (gpointer, gchar const**, gconstpointer*, guint32*);
-		}
-		kv;
+			gboolean (*backend_get_all)(gchar const*, gpointer*);
+			gboolean (*backend_get_by_prefix)(gchar const*, gchar const*, gpointer*);
+			gboolean (*backend_iterate)(gpointer, gchar const**, gconstpointer*, guint32*);
+		} kv;
```

Comment:
Please do not change the formatting of unrelated code, this makes reviewing unnecessarily complicated.

Reply:
I use an automated code-formatting tool. It is possible to restrict a tool to a file, but not to individual lines. I will remove those space-only changes.